WordPress/src/main/res/values/strings.xml

```diff
@@ -1086,9 +1086,9 @@
     <string name="scan_fixing_threats_description">We\'re hard at work fixing these threats in the background. In the meantime feel free to continue to use your site as normal, you can check the progress at anytime.</string>
     <string name="scan_scanning_is_initial_description">Welcome to Jetpack Scan, we are taking a first look at your site now and the results will be with you soon.</string>
     <string name="scan_provisioning_description">Welcome to Jetpack Scan! We\'re scoping out your site, setting up to do a full scan. We\'ll let you know if we spot any issues that might impact a scan, then your first full scan will start.</string>
-    <string name="scan_idle_manual_scan_description">Run a manual scan now or wait for Jetpack to scan your site later today.</string>
-    <string name="scan_idle_last_scan_description">The last Jetpack scan ran %s and everything looked great. %s</string>
-    <string name="scan_idle_with_threats_description">The scan found %1s potential threats on %2s. Please review them below and take action. We\'re %3$s if you need us.</string>
+    <string name="scan_idle_manual_scan_description">To review your site again run a manual scan, or wait for Jetpack to scan your site later today.</string>
+    <string name="scan_idle_last_scan_description">The last Jetpack scan ran %s and did not find any risks. %s</string>
+    <string name="scan_idle_with_threats_description">The scan found %1s potential threats with %2s. Please review them below and take action or tap the fix all button. We\'re %3$s if you need us.</string>
```

Comment:
Question (❓): Is `We\'re` what we prefer to use instead of `We are` that is mentioned in the copy improvements?

Reply:
I think the copy improvements mentioned  `We are` because it was the same in the original message. `We\'re` was copied from the Calypso message and I think copy improvements was focussed more on including fix all button tapping directions.
However I changed it to "We are" in ed20c7c to be consistent with iOS. Thanks for noticing it 🌟.

Reply:
Thanks for the clarification @ashiagr ! 🌟 

Yeap, I was actually doing blindly a copy review based on what I was having at hand, I wasn't sure about the details.